### DISCUSSION TITLE
Multiple choice question UI is not displayed correctly in apk

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#668] [General Discussion]
Using SDK 33, I have installed the release 4.6 apk.

Any course's user interface for assessment is improper. 
We are unable to view the entire question because the questions are being trimmed.

![image (2)](https://github.com/Sunbird-Ed/Community/assets/72926048/c8f96865-57a6-4df0-90c1-7afad21c486f)
![WhatsApp Image 2023-12-07 at 08 52 45](https://github.com/Sunbird-Ed/Community/assets/72926048/bad6e8bc-5c78-49e7-a7a3-86010ab505e9)

Is there any solution for this issue.

cc: @swayangjit @vpPavithra @luckynvdu @Krishnaj20

**Comment by swayangjit** [comment]
@siddhishinde0723 Please follow this [thread](https://github.com/orgs/Sunbird-Knowlg/discussions/184)

**Reply by vinukumar-vs** [reply]
[siddhishinde0723](https://github.com/siddhishinde0723) The intention of the templates to solve different purpose. We are not supporting or doing any active development on Content-Player (v1). Because it is updated tech stack and hard to setup as the dependant libraries support for NODE etc.

Hence any changes required on this would suggest to clone to content-plugins repo and build it on your own. 

@ashokreddy1208 @swayangjit @mohitga
